modules/allocating-additional-resources-to-data-science-users.adoc

```diff
@@ -6,19 +6,14 @@
 [role='_abstract']
 As a cluster administrator, you can allocate additional resources to a cluster to support compute-intensive data science work. This support includes increasing the number of nodes in the cluster and changing the cluster's allocated machine pool.
 
-ifdef::upstream[]
+ifdef::upstream,self-managed[]
 For more information about allocating additional resources to an {openshift-platform} cluster, see link:https://docs.openshift.com/container-platform/{ocp-latest-version}/machine_management/manually-scaling-machineset.html[Manually scaling a compute machine set].
 endif::[]
 
-ifndef::upstream[]
-ifdef::self-managed[]
-For more information about allocating additional resources to an {openshift-platform} cluster, see link:https://docs.openshift.com/container-platform/{ocp-latest-version}/machine_management/manually-scaling-machineset.html[Manually scaling a compute machine set].
-endif::[]
-
-ifndef::self-managed[]
+ifdef::cloud-service[]
 .Prerequisites
 * You have credentials for administering clusters in OpenShift Cluster Manager (link:https://console.redhat.com/openshift/[https://console.redhat.com/openshift/]). For more information about configuring administrative access in OpenShift Cluster Manager, see link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2023/html-single/managing_clusters/index#assembly-user-management-ocm[Configuring access to clusters in OpenShift Cluster Manager].
-* If you want to increase the size of a machine pool by using accelerators, your {openshift-platform} cluster supports them.
+* If you want to increase the size of a machine pool by using accelerators, your OpenShift cluster supports them.
```

Comment:
I know it's unrelated to your change, but I don't love the way that this prereq is written.

Suggest: "If you intend to to increase the size of a machine pool by using accelerators, you have ensured that your OpenShift cluster supports them."

Reply:
Also, just double-checking: it looks like this whole procedure applies only to the cloud service. Is that the intention?